packages/frontend/src/components/ProjectConnection/WarehouseForms/defaultValues.ts

```diff
@@ -104,7 +104,7 @@ export const SnowflakeDefaultValues: CreateSnowflakeCredentials = {
     queryTag: '',
     accessUrl: '',
     startOfWeek: undefined,
-    authenticationType: 'private_key',
+    authenticationType: undefined,
```

Comment:
We want to handle this in the component rather than refactor how we load default values into the Warehouse forms.